docs/upgrade.md

```diff
@@ -172,38 +172,31 @@ After some time, you should see that new Istio pods are running:
 ```bash
 $ kubectl get pods -n istio-system --watch
 NAME                                      READY     STATUS    RESTARTS   AGE
-istio-citadel-7664c58768-l8zgb            1/1       Running   0          7m
-istio-egressgateway-8588c7c8d-wkpgk       1/1       Running   0          7m
-istio-galley-78b8467b4d-b5dqs             1/1       Running   0          7m
 istio-ingressgateway-5c48b96cb4-lnfsn     1/1       Running   0          7m
 istio-operator-controller-manager-0       2/2       Running   0          16m
-istio-pilot-84588fff4c-4lhq8              2/2       Running   0          7m
-istio-policy-75f84689f5-78dxr             2/2       Running   0          7m
-istio-sidecar-injector-66cd99d8c8-bp4j7   1/1       Running   0          7m
-istio-telemetry-7b667c5fbb-2lfdc          2/2       Running   0          7m
+istiod-84588fff4c-4lhq8                   2/2       Running   0          7m
```

Comment:
/lgtm 
for fixing the doc

Reply:
In the end you have an istiod pod instead of the four components(Citadel、Galley、Pilot and the Sidecar Injector). Please see this [istiod](https://banzaicloud.com/blog/istio-operator-1.5/) 、[Mixerless telemetry or Telemetry V2](https://banzaicloud.com/blog/istio-mixerless-telemetry/) and [istio-multiple-gateways](https://banzaicloud.com/blog/istio-multiple-gateways/) if you are interested in this.

add some explanation is better for me.

Reply:
Explanation added.